dmoj/graders/bridged.py

```diff
@@ -14,9 +14,14 @@ class BridgedInteractiveGrader(StandardGrader):
     def __init__(self, judge, problem, language, source):
         super().__init__(judge, problem, language, source)
         self.handler_data = self.problem.config.interactive
-        self.interactor_binary = self._generate_interactor_binary()
-        self.contrib_type = self.handler_data.get('type', 'default')
 
+        try:
+            self.interactor_binary = self._generate_interactor_binary()
+        except CompileError as compilation_error:
+            # Rethrow as IE to differentiate from the user's submission failing to compile.
+            raise InternalError('Interactor failed compiling') from compilation_error
```

Comment:
Nit for consistency with most other errors
```suggestion
            raise InternalError('interactor failed compiling') from compilation_error
```